### PR TITLE
9 create db for storage of warehouse

### PR DESCRIPTION
**Pull request by davidebschke**
Fix the dynamoDB table and add an example Item to show all is correct. The item is for example ok that is inside

### CODE REVIEW

File: terraform/dynamodbwarehouse.tf [hunk 1]
```diff
@@ -1,10 +1,9 @@
-resource "aws_dynamodb_table" "basic-dynamodb-table" {
+resource "aws_dynamodb_table" "WarehouseDB" {
```
Comment (fabianschmauder) [inline terraform/dynamodbwarehouse.tf]:
```suggestion
resource "aws_dynamodb_table" "warehouseDB" {
```

lower case ;)

File: terraform/dynamodbwarehouse.tf [hunk 2]
```diff
@@ -16,33 +15,26 @@ resource "aws_dynamodb_table" "basic-dynamodb-table" {
...
-    non_key_attributes = ["description"]
-  }
+global_secondary_index {
+    name = "IndexForGoodsName"
+    hash_key = "Goods_Name"
+    write_capacity = 10
+    read_capacity = 10
+    projection_type = "INCLUDE"
+    non_key_attributes = ["Amount_of_goods", "price_of_goods"]
+    }
```
Comment (fabianschmauder) [inline terraform/dynamodbwarehouse.tf]: lower keys names and keys ;)

File: terraform/dynamodbwarehouse.tf [hunk 3]
```diff
@@ -16,33 +15,26 @@ resource "aws_dynamodb_table" "basic-dynamodb-table" {
...
-  global_secondary_index {
-    name               = "IndexForPriceOfGoods"
-    hash_key           = "Goods_Name"
-    range_key          = "Price_of_goods"
-    write_capacity     = 10
-    read_capacity      = 10
-    projection_type    = "INCLUDE"
-    non_key_attributes = ["description"]
+# Example Item to check the Table
+resource "aws_dynamodb_table_item" "warehouse_item_example" {
```
Comment (fabianschmauder) [inline terraform/dynamodbwarehouse.tf:29]: do you still need this?

**Review by fabianschmauder — COMMENTED** [review]
just change your naming to lower case ;)

File: terraform/dynamodbwarehouse.tf (hunk 3 above)
Comment (davidebschke) [reply terraform/dynamodbwarehouse.tf:29]: Yes that will be delete later for the first time i need this

**Pull request merged**